src/plugins/telemetry/server/fetcher.ts

```diff
@@ -137,6 +202,13 @@ export class FetcherTask {
       telemetrySavedObject,
     });
 
+    const lastReported = telemetrySavedObject ? telemetrySavedObject.lastReported : void 0;
+
+    // If the lastReported value in the SO is more recent than the in-memory one, refresh the memory (another instance or the browser may have reported it)
+    if (lastReported && lastReported > this.lastReported$.value) {
+      this.lastReported$.next(lastReported);
+    }
```

Comment:
This is mostly to avoid retrying every minute during a whole day if another instance reported earlier that same day.